lib/basedriver/helpers.js

```diff
@@ -202,6 +208,53 @@ async function testZipArchive (zipPath) {
   }
 }
 
+async function copyFromWindowsNetworkShare (app, appExt, mountroot, windowsUserName, windowsPassword) {
+  if (system.isWindows()) {
+    return await copyLocallyFromWindowsShare(app, appExt);
+  } else {
+    return await mountWindowsShareOnMac(app, mountroot, windowsUserName, windowsPassword);
+  }
+}
+
+async function mountWindowsShareOnMac (app, mountroot, windowsUserName, windowsPassword) {
+  let pathSplit = app.split("\\");
+  let networkShare = pathSplit[2];
+  let rootFolder = pathSplit[3];
+  app = app.replace(/\\/g, "/");
+  app = app.replace(`/${networkShare}`, mountroot);
+  let mountPath = `/${mountroot}/${rootFolder}`;
+
+  let mountNetworkShare = async function () {
+    await fs.mkdir(mountPath);
+    let mountArgs = [`-t`, `smbfs`, `${windowsUserName}:${windowsPassword}@${networkShare}/${rootFolder}`, mountPath];
+    try {
+      await exec('mount', mountArgs, { maxBuffer: 524288 });
```

Comment:
we don't need to specify the `maxBuffer` option since we're using `teen_process`'s exec. it will have no effect.